### PR TITLE
Fixes from QA - batch 6

### PR DESCRIPTION
**Pull request by e-kolpakov**
**Depends on:** https://github.com/edx-solutions/XBlock/pull/6

**Testing instructions:**
1. TA with staff access:
1.1. Grant TA company or internal admin role. 
1.2. Make sure there's at least one TA graded activity (Min required reviews = 0 in studio for activity) with Peer Review stage. 
1.3. Go to Admin -> Group  Work -> Select Course -> Group Work Status -> Choose any group -> Click "TA Graded" in any column.
1.4. You should arrive to Group Project at selected stage. **Expected result:** Can grade group and submit feedback. **Failure condition:** clicking submit does not post feedback.
2. Copy Update - after submitting any upload, the confirmation dialog text should not contain "passes" at the end.
3. Review stages state
3.1. Review stage (both Peer Grading and team Evaluation) stages should not be marked as complete or partially complete unless student visited them first when they were opened.
4. Preserving question IDs over export-import
4.1. Create some Review Question XBlocks, but **do not edit Question ID** (or revert to default, if accidentally edited).
4.2. Add some Evaluation Display or Grade Display XBlocks to appropriate stages (Team Evaluation or Grade Display stages, respectively). Edit them and choose any question available. Make sure there are no validation error.
4.3. Export the course (depends on https://github.com/edx-solutions/XBlock/pull/6)
4.4. Import the course (as a new course or replacing existing course).
4.5. Navigate to Team Eval or Grade Display stage in Studio. **Expected result:** still no validation errors, questions are still selected. **Failure condition:** questions are not selected for Evaluation/Grade Display XBlocks, validation errors appear.
5. TA has access to Resources and Discussion views
5.1. Navigate to Group Project in TA grading mode (see 1.3)
5.2. Observe that Project Navigator shows Resources and Private Discussion selectors. 
5.3. Clicking the selectors should activate corresponding views.

### CODE REVIEW

**Comment by Kelketek** [comment]
@e-kolpakov Issue 1 still tails with:

```
2015-11-19 09:24:26,061 ERROR 23191 [django.request] base.py:213 - Internal Server Error: /courses/Group/Work/Redux/xblock/i4x:;_;_Group;_Work;_gp-v2-stage-peer-review;_b1f0afdc952b47b0a6417721a8a1cf66/handler/submit_review
Traceback (most recent call last):
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 109, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/edx/app/edxapp/edx-platform/lms/djangoapps/courseware/module_render.py", line 911, in handle_xblock_callback
    return _invoke_xblock_handler(request, course_id, usage_id, handler, suffix, course=course)
  File "/edx/app/edxapp/edx-platform/lms/djangoapps/courseware/module_render.py", line 1032, in _invoke_xblock_handler
    resp = instance.handle(handler, req, suffix)
  File "/edx/app/edxapp/xblocks/XBlock/xblock/mixins.py", line 86, in handle
    return self.runtime.handle(self, handler_name, request, suffix)
  File "/edx/app/edxapp/edx-platform/common/lib/xmodule/xmodule/x_module.py", line 1270, in handle
    return super(MetricsMixin, self).handle(block, handler_name, request, suffix=suffix)
  File "/edx/app/edxapp/xblocks/XBlock/xblock/runtime.py", line 1025, in handle
    results = handler(request, suffix)
  File "/edx/app/edxapp/xblocks/XBlock/xblock/mixins.py", line 65, in wrapper
    response = func(self, request_json, suffix)
  File "/edx/app/edxapp/xblocks/xblock-group-project-v2/group_project_v2/utils.py", line 127, in wrapper
    return func(*args, **kwargs)
  File "/edx/app/edxapp/xblocks/xblock-group-project-v2/group_project_v2/utils.py", line 141, in wrapper
    return func(*args, **kwargs)
  File "/edx/app/edxapp/xblocks/xblock-group-project-v2/group_project_v2/utils.py", line 153, in wrapper
    return func(*args, **kwargs)
  File "/edx/app/edxapp/xblocks/xblock-group-project-v2/group_project_v2/stage/review.py", line 125, in submit_review
    self.do_submit_review(submissions)
  File "/edx/app/edxapp/xblocks/xblock-group-project-v2/group_project_v2/stage/review.py", line 376, in do_submit_review
    self.activity.calculate_and_send_grade(group_id)
  File "/edx/app/edxapp/xblocks/xblock-group-project-v2/group_project_v2/group_project.py", line 455, in calculate_and_send_grade
    'max_value': self.weight,
  File "/edx/app/edxapp/edx-platform/lms/djangoapps/courseware/module_render.py", line 520, in publish
    handle_grade_event(block, event_type, event)
  File "/edx/app/edxapp/edx-platform/lms/djangoapps/courseware/module_render.py", line 455, in handle_grade_event
    max_grade,
  File "/edx/app/edxapp/edx-platform/lms/djangoapps/courseware/model_data.py", line 932, in set_score
    assert user_id == self.user.id
AssertionError
```

 ...even with the export you provided.

You have a linter failure.

Everything else seems to work, checking over the code itself now.

File: group_project_v2/templates/html/stages/team_evaluation.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<form class="review peer_review" data-review-type="peer_review" action="submit_review" method="POST">
+<div class="review peer_review" data-review-type="peer_review" data-action="submit_review" data-method="POST">
```
Comment (Kelketek) [inline group_project_v2/templates/html/stages/team_evaluation.html]: Why is this needed? Can it be avoided? This is an accessibility headache. I know we don't have to pass accessibility requirements for this block right now, but it's likely going to have to be undone if we ever intend to see this get wider use.

Comment (e-kolpakov) [inline group_project_v2/templates/html/stages/team_evaluation.html]: @Kelketek LMS appends some staff-only content in a `section` tag. And section is not allowed inside a form, or, at least, it terminates form right after itself in Chrome. So we can't have a form with multiple XBlocks in it. When we get to making it a wider use we'll have to address a lot more serious concerns (i.e. API, pluggable services, XBlock as a course tab, etc.)

**Comment by e-kolpakov** [comment]
@Kelketek what are `user_id` and `self.user.id` in that case? Which of htose is TA?

**Comment by e-kolpakov** [comment]
@Kelketek I wonder what have I done different to not cause the assertion error you saw. Anyway, it should be fixed now.

**Comment by Kelketek** [comment]
:+1:
